index.html

```diff
@@ -5,11 +5,152 @@
     <title>TALART CONFERENCE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,300&display=swap" rel="stylesheet">
+    <!--mozilla font-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Zilla+Slab:wght@500&display=swap" rel="stylesheet">
+    <!--lato-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <!--font awesome-->
+    <script src="https://kit.fontawesome.com/de9dd03ce2.js"crossorigin="anonymous"></script>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>WELCOME TO TALART. LET RECOGNIZE TALENTS IN THE ART INDUSTRY.</h1>
-    <p class="add" id="adds"></p>
+    <div class="wrapper">
+        <section class="navigation">
+            <div class="headnavss">
+                <ul class="topass">
+                    <li class="topitem"><i class="first-nav-bar-icons fa-brands fa-facebook-f"></i></li>
+                    <li class="topitem"><i class="first-nav-bar-icons fa-brands fa-twitter"></i></li>
+                    <li class="topitem">French</li>
+                    <li class="topitem">mypage</li>
+                    <li class="topitem">Logout</li>
+                </ul>
+            </div>
+            <div class="nav">
+            <div class="logo">
+                <a href="index.html" class="logss">TAL-ART</a>
+            </div>
+            <ul class="navlist">
+                <li><a href="index.html" class="active">Home</a></li>
+                <li class="navitems"><a href="about.html">About</a></li>
+                <li class="navitems"><a href="">Leagues</a></li>
+                <li class="navitems"><a href="">Reach out</a></li>
+            </ul>
+        </div>
+        </section>
+
+        
+        <ul class="navvme">
+            <li class="cloze"><a href="#" class="close"><img src="asset/image/close.png" alt="closebtn"></a></li>
+            <li class="navitemx"><a href="index.html" class="navlinksx">Home</a></li>
+            <li class="navitemx"><a href="about.html" class="navlinksx">About</a></li>
+            <li class="navitemx"><a href="#contact" class="navlinksx">Contact</a></li>
+        </ul>
+
+        <!-----header----->
+        <section class="header">
+            <div class="headcontain">
+                <a href="#"><img src="asset/image/menu.png" alt="hamburger" class="hamburger"></a>
+                <h1 class="intro">"Hello! Artistic World"</h1>
+                <h1 class="message">TAL-ART INTERNATIONAL CONFERENCE 2022</h1>
```

Comment:
- [ ] Please There's no more than 1, `<h1>` tag on each page. because the `<h1>` tag stands for the most important heading of the web page. Kindly use this [link to learn more about](https://www.w3docs.com/learn-html/html-headings.html) `<h>` tags